portfolio/src/main/webapp/script.js

```diff
@@ -45,3 +45,20 @@ async function getMessage() {
     const message = "" + json.msg1 + " " + json.msg2 + " " + json.msg3;
     document.getElementById('message-container').innerText = message;
 }
+
+/**
+ *Gets the comments posted and prints them back
+ */
+async function getComments() {
+    const response = await fetch('/data');
+    const json = await response.json();
+    const posted_comments = document.getElementById('posted_comments');
+    const num_comments = json.num_comments;
+    const comments = json.comments;
+    var i;
```

Comment:
Try to reduce the scope of your variables. In this case, you could declare it inside the loop, i.e., `for (var i =`. (Though I think you are better off using `for...of`.)